packages/web-components/fast-html/src/fixtures/partial/partial.fixture.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title></title>
+        <script type="module" src="/partial/main.js"></script>
+    </head>
+    <body>
+        <f-template name="test-element">
+            <template>
+                <f-partial id="test">
```

Comment:
I still don't understand partial, i will reach out to you later for 1:1 training on it

Reply:
From my understanding, it is so that you can build out reusable templates to be composed other locations of the larger component template. In this example it looks like the partial is being used to build out a recursive template. The equivalent imperative code would probably look something like:

```ts
const itemTemplate = html`
  <li>
    ${x => x.text}
    ${when(x => x.items.length, html`
      <ul>
        ${repeat(x => x.items, () => itemTemplate)}
      </ul>
    `)}
  </li>
`;

const template = html`
  <ul>
    ${repeat(x => x.items, () => itemTemplate)}
  </ul>
`;
```

Reply:
fwiw @janechu I wonder if it would be worthwhile to align naming in this package with what is in the tagged template helper already. The `html.partial` helper has a different purpose than the `f-partial` tag here where `f-partial` looks to just be creating a `ViewTemplate`.

Other than the naming confusion, this all looks great to me and I look forward to seeing these changes in the next release.

Reply:
Thanks! This is in `alpha` so we can toy with the naming, I do agree we have a purpose vs naming misalignment here, I think the next step, after throwing this at a few scenarios, is opening up the discussion on naming. I am not too happy with `f-partial` and `f-apply` and would like some suggestions.